stream_constants.h.diff

```diff
@@ -0,0 +1,18 @@
+--- a/xbmc/addons/kodi-dev-kit/include/kodi/c-api/addon-instance/inputstream/stream_constants.h
```

Comment:
What's this file used for?

Reply:
Seems to come in on rebase, Sorry!

Becomes today kicked out and think to add also on .gitignore a `/*.diff` below of `/*.patch` to prevent it on base folder in future.